javascript/gps-sub.js

```diff
@@ -0,0 +1,6 @@
+channel.subscribe('position', msg => {
+  map.showMarker({
```

Comment:
Same goes here, I would recommend logging statements here

Reply:
I disagree, for reasons above.